lte/gateway/c/core/oai/tasks/amf/amf_as.cpp

```diff
@@ -1191,11 +1191,6 @@ uint16_t amf_as_establish_cnf(
   // Setting-up the AS message
   as_msg->ue_id = msg->ue_id;
 
-  if (msg->pds_id.guti == NULL) {
-    OAILOG_WARNING(LOG_NAS_AMF, "AMFAS-SAP - GUTI is NULL...");
-    OAILOG_FUNC_RETURN(LOG_NAS_AMF, ret_val);
-  }
-
```

Comment:
why was this check here before and why is it removed now?

Reply:
The above code was committed from Chandra on April 12(#5984).
reviewed PR, above specific variable, is not initialized.

I am using the UERANSIM script with Registration & PDUSession, sometimes it was failing due to the above check.
backtraced pds_id.guti variable is unused & uninitilzed value.

Reply:
till now we did not see any issue because pds_id.guti value garbage, and amf not used this variable.

Reply:
basically, amf_sap structure is not initialized,  I have called memset all places where  amf_sap  defined

Reply:
function local variables initialization is a basic coding guideline, it was missed in base code.
I did not change any functionality and not added any UT cases.
 
Please provide your feedback,